source/sass/components/article-page.scss

```diff
@@ -409,11 +409,24 @@
   }
 }
 
+.article-navbar-container {
+  // Move the summary bar up 50px just to hide it.
+  // This then slides down to become visible
+  --top-offset: -50px;
+  position: relative;
+  top: var(--top-offset);
+  z-index: -1;
```

Comment:
```suggestion

```

Reply:
I applied this. I actually need to undo this. -1 z-index is required for proper layering here. 

![image](https://user-images.githubusercontent.com/4743971/102912601-d529a400-443a-11eb-9e0a-3e9402d9498a.png)

Reply:
ah interesting! i see it now!